op-batcher/batcher/driver.go

```diff
@@ -273,6 +273,14 @@ func (l *BatchSubmitter) loadBlocksIntoState(ctx context.Context, start, end uin
 			return err
 		}
 		latestBlock = block
+
+		if i%100 == 0 {
```

Comment:
1. The 100 block parameter could become a CLI config variable.
2. `i` is actually the block number, not an index into how many blocks were loaded so far. We could define a new counter to track actually blocks loaded instead.

Reply:
This feels ok to hard code. If this just checks if there is enough data loaded up before publishing, this is fine. If it forces a publish irregardless I'd prefer a larger value.